kornia/filters/sobel.py

```diff
@@ -117,19 +257,20 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:  # type: ignore
                        self.kernel.size(4) // 2,
                        self.kernel.size(4) // 2]
         out_ch: int = 6 if self.order == 2 else 3
-        return F.conv3d(F.pad(input, spatial_pad, 'replicate'), kernel, padding=0, groups=c).view(b, c, out_ch, d, h, w)
+        input_pad: torch.Tensor = F.pad(input, spatial_pad, 'replicate')
+        return F.conv3d(input_pad, kernel_flip, padding=0, groups=c).view(b, c, out_ch, d, h, w)
```

Comment:
@ducha-aiki I might found a bug here. The input of the conv3d had `kernel` instead of `kernel_flip`